src/helpers/initialize.js

```diff
@@ -11,13 +11,14 @@ function initialize({ component, player, playerOpts }) {
   player.on('error', component.eventHandlers.onError);
   player.on('adPlay', component.eventHandlers.onAdPlay);
   player.on('adPause', component.props.onAdPause);
+  player.on('adComplete', component.props.onAdComplete);
+  player.on('adTime', component.eventHandlers.onTime);
```

Comment:
`on-ad-time.js` would use the same logic as `on-time.js`, so I thought I would use the same code.  However, since others that are using the package may be relying on the existing logic, I agree that separating is better.